### PR TITLE
refact!: use dataset IDs in experiment API filters + VCF2MAF fixes

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: chord_metadata_service/experiments/tests/test_api.py [hunk 1]
```diff
@@ -58,95 +61,57 @@ def test_get_experiment_schema(self):
...
         response = self.client.get('/api/experimentresults?file_format=vcf')
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        response_data = response.json()
-        self.assertEqual(response_data["count"], 2)
-        self.assertEqual(len(response_data["results"]), 2)
+        self.assert_response_200_and_length(response, 2)
 
     def test_filter_experiment_results_url(self):
         response = self.client.get('/api/experimentresults?url=example.org')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (v-rocheleau) [inline chord_metadata_service/experiments/tests/test_api.py]: Can be removed

File: chord_metadata_service/chord/workflows/wdls/vcf2maf.wdl [hunk 2]
```diff
@@ -180,40 +191,59 @@ task vcf_2_maf {
...
+                --tmp-dir . \
+                --verbose \
                 --vep-data ~{vep_cache_dir} \
+                --species ${VEP_CACHE_PATH_SPECIES} \
                 --ref-fasta ${REF_FASTA_PATH}/${REF_FASTA_TOPLEVEL} \
+                --ncbi-build ${assembly_id} \
                 --vep-path ${VEP_PATH} \
-                --tumor-id ${SAMPLE_ID}
+                --tumor-id ${TUMOR_ID} \
+                --normal-id ${NORMAL_ID}
```
Comment (v-rocheleau) [inline chord_metadata_service/chord/workflows/wdls/vcf2maf.wdl:230]: Should these be changed to `~{}` ?

**Review by v-rocheleau — CHANGES_REQUESTED** [review]
Looks good, small comments

File: chord_metadata_service/chord/workflows/wdls/vcf2maf.wdl (hunk 2 above)
Comment (davidlougheed) [reply chord_metadata_service/chord/workflows/wdls/vcf2maf.wdl:230]: they're references to env vars inside the wdl script, not WDL variables

**Review by v-rocheleau — APPROVED** [review]
lgtm

**Pull request merged**